build/append-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+printf "TESTS=" >> Makefile.am
+
+T=$(find test/test-cases -iname *.json)
```

Comment:
I don't care one way or another if the test names are on a single line or multiple lines. IMO if it's an autogenerated list it doesn't matter.

Putting the Bash into the `Makefile.am`, as it is in https://github.com/SpiderLabs/ModSecurity/blob/v3/master/Makefile.am#L87, doesn't work for obvious reasons. So I decided to put it into `build.sh`. Let me know if you have an idea for a better place. I am not overly familiar with automake tools in this regard.

This PR is really only about getting all the appropriate tests to run when `make test` and Travis are executed, so that authors don't have to manually add file names to `Makefile.am`. I have no preference for how things are formatted, only that it's done. Let me know your thoughts on how best to move forward so we see all tests being executed consistently.

Reply:
That affects the ability of the users to manipulate the generated-distributed files. I understand that you made the patch targeting a very specific point, but it may lead to a negative impact on others. In order to have the code in the mainline we need to make sure that it is not causing a negative impact to any user.

Reply:
How does this impact other users? Is your point that you want each test definition on a separate line? That's completely fine with me, as I already mentioned. The only goal here is to ensure that the tests that were written are actually running. There are already a fair number of tests that are not being executed by Travis/`make check`.